### DISCUSSION TITLE
Matching LOGP and GP-log

### DISCUSSION THREAD

**Discussion by MarGoProHki** [gopro/labs#684] [Q&A]
I’m trying to match Hero 11 (with Labs) to Hero 12’s GP-log. What would be the best LOGP values and are there some other settings to change?

**Comment by dnewman-gpsw** [comment]
On HERO11 set mVcFoMLOGB=400oMWIDE=1 This is the same a GP-Log on HERO12.

![image](https://github.com/gopro/labs/assets/10856084/efad99d4-9882-4652-b655-fdfa6c8d4914)

mV - mode Video
cF - color Flat
oMLOGB=400 - Log base curve 400
oMWIDE=1 - use the wide gamut

Permanent version is mVcF!MLOGB=400!MWIDE=1

When using Labs with either LOGB or GP-Log on HERO12, use an higher bitrate oMBITR=160 and decrease the amount of in-camera noise reduction (to taste) start with oMNR01=50 (or lower.)

**Reply by MarGoProHki** [reply]
Thank you!

**Reply by MarGoProHki** [reply]
I tried values on H11 and compared to H12. LOGP400 and WIDE=1 isn’t exact match. Need to do some testing but I believe the truth is somewhere between 420 and 450.

**Reply by dnewman-gpsw** [reply]
Interesting.  It could be as Labs implementations and mainline FW are likely different. I will do some testing.

**Reply by dnewman-gpsw** [reply]
I ran some scripts on a HERO12 to switch between GP-Log and oMLOGB=400oMWIDE=1 captures, and found them to be the same.  There could be some under the hood tuning differences between HERO11 and HERO12 (black level might be different,) that makes the curve look different (or just camera to camera difference in sensor performance,) but the curve implementation between Labs and stock firmware is the same in my tests.  At first I thought there was a difference, but I was using GP-Log defaults with ISO 100-400 range, and I was comparing indoor light, with Labs LOGB=400 with a 100-1600 range, the noise changes made the curve look a little different.

**Comment by MarGoProHki** [comment]
Ok, good to know. Thanks. For some reason H12 GP-log just looks different and some how better, but H11 with LOGP and WIDE isn’t far behind.

Labs and QRcontrol app is so much fun to experiment. I finally found settings to get rid off hypersmooth jerkines on low light. With Labs H11 is good enough for my needs. I’m returning H12 to shop.

**Reply by dnewman-gpsw** [reply]
Hmmm.  Don't say that.  Labs should be a positive thing, shouldn't be a reason to return a camera.  Reason to keep:  historically newer cameras are developed longer, and HERO12 is the current stability king, I will only recommend of H12 for critical long form Labs scripting.

**Reply by MarGoProHki** [reply]
Nothing against H12, it’s great camera. The reason is I’ve been using H11 a lot and I have three of those at the moment. It’s more convenient to use same generation cameras if there are image quality differences between generations, and that’s the case with H11 and H12. It makes no sense to upgrade all of my H11s. Also, I like H11 UI more and it feels faster.
